e2e/__tests__/__snapshots__/v8Coverage.test.ts.snap

```diff
@@ -9,8 +9,8 @@ exports[`on node >=10 prints coverage 1`] = `
 ----------|---------|----------|---------|---------|-------------------
 File      | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
 ----------|---------|----------|---------|---------|-------------------
-All files |     100 |      100 |     100 |     100 |                   
- Thing.js |     100 |      100 |     100 |     100 |                   
+All files |     100 |      100 |       0 |     100 |                   
+ Thing.js |     100 |      100 |       0 |     100 |                   
```

Comment:
this is weird... I can play with it locally and see if I understand it

Reply:
hmm, I cannot reproduce this. It's still `100` on my machine, both on master and this branch

Reply:
wait, I get the same change when running the e2e test, but not if I run jest manually within the fixture

Reply:
Okay, so with my changes it's fixed and I can change the snapshot back I think? I must have had something cached

Reply:
yeah, seems like it. I pushed a revert of the snapshot update, let's see what CI says